### PR TITLE
🔀 :: API Connect

### PR DESCRIPTION
**Pull request by baekteun**
## 개요
각 Reactor에 DI
통신 Request

## 작업사항
- 모든 Reactor에 UseCase Injection
- 실제 Request

### CODE REVIEW

**Review by kimsh153 — CHANGES_REQUESTED** [review]
LGTM

File: Service/Sources/Data/DataSource/Remote/DataSource/BaseRemote.swift [hunk 1]
```diff
@@ -7,7 +7,7 @@ import AVFoundation
 class BaseRemote<API: GCMSAPI> {
     public var testStatus = false
     public var successStatus = true
-    private let provider = MoyaProvider<API>(plugins: [JWTPlugin()])
+    private let provider = MoyaProvider<API>(plugins: [JWTPlugin(), NetworkLoggerPlugin()])
```
[the remark this answers is not in the document]
Comment (baekteun) [reply Service/Sources/Data/DataSource/Remote/DataSource/BaseRemote.swift]: "느려." [1edbe6](https://github.com/GSM-MSG/GCMS-iOS/pull/71/commits/1edbe6251a4a47dd5161098855ce8876a4f3b3a1)

File: iOS/Sources/Presentation/Scene/Main/DetailClub/View/HeaderLabel.swift [hunk 2]
```diff
@@ -11,4 +11,8 @@ final class HeaderLabel: UILabel {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    public func setTitle(title: String?) {
+        self.text = title
+    }
```
Comment (kimsh153) [inline iOS/Sources/Presentation/Scene/Main/DetailClub/View/HeaderLabel.swift:17]: 바꿔

**Review by kimsh153 — CHANGES_REQUESTED** [review]

File: iOS/Sources/Presentation/Scene/Main/DetailClub/View/HeaderLabel.swift (hunk 2 above)
Comment (baekteun) [reply iOS/Sources/Presentation/Scene/Main/DetailClub/View/HeaderLabel.swift]: 수정! [943a18](https://github.com/GSM-MSG/GCMS-iOS/pull/71/commits/943a18c53427cf6f27528673f46be47200531227)

**Review by chanwoo00106 — APPROVED** [review]
어디선가 들려오는 pr의 소리
잘 먹겠습니다

**Review by khs3994 — APPROVED** [review]

**Review by kimsh153 — APPROVED** [review]
LGTM

**Review by kimsh153 — APPROVED** [review]

**Review by yoonjibin — APPROVED** [review]

**Review by Huuuunee — APPROVED** [review]

**Review by khs3994 — APPROVED** [review]

**Pull request merged**